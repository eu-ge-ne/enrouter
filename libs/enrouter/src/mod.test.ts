import { describe, test, expect } from "vitest";

import { hello } from "./mod.js";

describe("enrouter", () => {
  test("hello", async () => {
    expect(hello()).toMatchSnapshot();
  });
});
