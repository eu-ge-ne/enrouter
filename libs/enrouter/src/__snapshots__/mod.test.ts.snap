// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`enrouter > hello 1`] = `"Hello enrouter"`;
